services/core-web/src/components/mine/Summary/MineSummary.tsx

```diff
@@ -0,0 +1,289 @@
+import { formatDate } from "@common/utils/helpers";
+import { getPartyRelationships } from "@mds/common/redux/selectors/partiesSelectors";
+import { getPartyRelationshipTypes } from "@mds/common/redux/selectors/staticContentSelectors";
+import { getMineComplianceInfo } from "@mds/common/redux/selectors/complianceSelectors";
+import { getMines } from "@mds/common/redux/selectors/mineSelectors";
+import { getPermits } from "@mds/common/redux/selectors/permitSelectors";
+import * as String from "@mds/common/constants/strings";
+import { Contact } from "@/components/mine/ContactInfo/PartyRelationships/Contact";
+import * as router from "@/constants/routes";
+import PermitCard from "@/components/mine/Permit/MinePermitCard";
+import { TSFCard } from "@/components/mine/Tailings/MineTSFCard";
+import { DOC, OVERDUEDOC } from "@/constants/assets";
+import MineHeader from "@/components/mine/MineHeader";
+import MineWorkInformation from "@/components/mine/Summary/MineWorkInformation";
+import {
+  IMatch,
+  IMine,
+  IPartyRelationshipType,
+  IPermitPartyRelationship,
+  IMineComplianceInfo,
+  IPermit,
+} from "@mds/common";
+
+/**
+ * @class MineSummary.js contains all content located under the 'Summary' tab on the MineDashboard.
+ */
+
+interface MineSummaryProps {
+  match: IMatch;
+  mines: Partial<IMine>;
+  partyRelationshipTypes: IPartyRelationshipType[];
+  partyRelationships: IPermitPartyRelationship[];
+  mineComplianceInfo: IMineComplianceInfo;
+  minePermits: Partial<IPermit>[];
+}
+
+const renderPartyRelationship = (mine, permits, partyRelationship, partyRelationshipTypes) => {
+  if (partyRelationshipTypes.length === 0) return <div>{String.LOADING}</div>;
+
+  const partyRelationshipTitle = partyRelationshipTypes.find(
+    ({ mine_party_appt_type_code }) =>
+      mine_party_appt_type_code === partyRelationship.mine_party_appt_type_code
+  ).description;
+
+  return (
+    <Col md={24} lg={12} xl={8} xxl={6} key={partyRelationship.mine_party_appt_guid}>
+      <Contact
+        mine={mine}
+        permits={permits}
+        partyRelationship={partyRelationship}
+        partyRelationshipTitle={partyRelationshipTitle}
+        compact
+      />
+    </Col>
+  );
+};
+
+const renderSummaryPermit = (permit, partyRelationships) => (
+  <Col sm={24} md={12} lg={8} xl={6} xxl={4} key={permit.permit_guid}>
+    <PermitCard permit={permit} PartyRelationships={partyRelationships} />
+  </Col>
+);
+
+const renderSummaryTSF = (tsf, partyRelationships) => (
+  <Col sm={24} md={12} lg={8} xl={6} xxl={4} key={tsf.mine_tailings_storage_facility_guid}>
+    <TSFCard tailingsStorageFacility={tsf} PartyRelationships={partyRelationships} />
+  </Col>
+);
+
+// Since end date is stored at yyyy-mm-dd, comparing current Date() to
+// the the start of the next day ensures appointments ending today are displayed.
+const isActive = (pr) =>
+  (!pr.end_date ||
+    moment(pr.end_date)
+      .add(1, "days")
+      .toDate() > new Date()) &&
```

Comment:
we don't need to convert moment to a date.

moment(date).add(1 days).isBefore() <-- isBefore used without an argument will compare to the equivalent of new Date(). There's a few variations on that function like isAfter if I'm getting my < and > mixed up. Would do something similar below with the Date.parse.

Reply:
Nice! I made the change it looks a lot cleaner now.